server/controllers/LibraryItemController.js

```diff
@@ -738,6 +738,8 @@ class LibraryItemController {
       return res.sendStatus(404)
     }
     const ebookFilePath = ebookFile.metadata.path
+    
+    Logger.info(`[LibraryItemController] User "${req.user.username}" requested download for item "${ebookFile.metadata.title}" at "${ebookFilePath}"`)
```

Comment:
```suggestion
    Logger.info(`[LibraryItemController] User "${req.user.username}" requested download for item "${req.libraryItem.media.metadata.title}" ebook at "${ebookFilePath}"`)
```
`title` doesn't exist on `ebookFile.metadata`